lib/install.js

```diff
@@ -7,7 +7,7 @@ import { system, tempDir, fs, logger, zip, mkdirp } from 'appium-support';
 
 const log = logger.getLogger('Chromedriver Install');
 
-const CD_VER = process.env.npm_config_chromedriver_version || process.env.CHROMEDRIVER_VERSION || '2.39';
+const CD_VER = process.env.npm_config_chromedriver_version || process.env.CHROMEDRIVER_VERSION || '2.40';
```

Comment:
can we take this value directly from CHROMEDRIVER_CHROME_MAPPING by just selecting the max semver out of its keys?

Reply:
Yeah, could do that.